src/v2/Apps/Authentication/authenticationRoutes.tsx

```diff
@@ -56,45 +58,65 @@ export const authenticationRoutes: AppRouteConfig[] = [
     },
   },
   {
-    path: "/login2",
+    path: "/login",
     hideNav: true,
     hideFooter: true,
     getComponent: () => LoginRoute,
-    onServerSideRender: runAuthMiddleware,
-    onClientSideRender: () => {
-      return LoginRoute.preload()
+    onServerSideRender: props => {
+      redirectIfLoggedIn(props)
+      runAuthMiddleware(props)
+    },
+    onClientSideRender: ({ match }) => {
+      setCookies(match.location.query)
+      LoginRoute.preload()
     },
   },
   {
-    path: "/log_in2",
+    path: "/log_in",
     render: ({ match }) => {
-      throw new RedirectException(`/login2${match.location.search}`, 301)
+      throw new RedirectException(`/login${match.location.search}`, 301)
     },
   },
   {
-    path: "/reset_password2",
+    path: "/reset_password",
     hideNav: true,
     hideFooter: true,
     getComponent: () => ResetPasswordRoute,
-    onServerSideRender: runAuthMiddleware,
-    onClientSideRender: () => {
-      return ResetPasswordRoute.preload()
+    onServerSideRender: ({ req, res }) => {
+      if (req.query.reset_password_token) {
+        req.session.reset_password_token = req.query.reset_password_token
+        req.session.set_password = req.query.set_password
+        req.session.reset_password_redirect_to =
+          req.query.reset_password_redirect_to
```

Comment:
None of this actually should matter. The new implementation only looks at the query string. This used to be a hack to get stuff into Sharify